lib/destroy.js

```diff
@@ -28,6 +42,6 @@ function destroy (state) {
 
     delete state.account
 
-    return clone(accountProperties)
-  })
+    return Promise.resolve(clone(accountProperties))
+  }
 }
```

Comment:
I can highly recommend this blog post for lots of interesting gotchas about Promises:
https://pouchdb.com/2015/05/18/we-have-a-problem-with-promises.html